rustls/Cargo.toml

```diff
@@ -17,6 +17,36 @@ build = "build.rs"
 [features]
 default = []
 
+# NOTE: This unstable & undocumented feature is intended to help make it easier to configure
+# docs.rs to generate the documentation for all features where this is needed,
+# excluding any potentially problematic features such as:
+# - `unsafe-assume-single-core`
+# - FIPS (if added back to this fork someday in the future)
+#
+# Keeping some other features in this list to help make developer testing with
+# almost all features a little easier.
+#
+# Developer testing with all features enabled except for `unsafe-assume-single-core` can be
+# approximated by: --features unstable-api-all,critical-section
+#
```

Comment:
I think there should be an explicit reference to the possibility of documenting & supporting this list for a crate user who may want to enable almost all possible features, say "full" or "full-std" for example. It looks like the use of "full" feature name for this kind of purpose has been done famously in the past:
- https://github.com/tokio-rs/tokio/blob/tokio-1.43.0/tokio/Cargo.toml#L29 - has `full` feature to "enable everything"
- this comment: https://github.com/rust-lang/cargo/issues/13936#issuecomment-2173301852
- https://www.reddit.com/r/rust/comments/yt6isn/is_there_a_known_way_to_minimize_features_of/?rdt=45920
- https://blog.jetbrains.com/clion/2020/10/intellij-rust-new-functionality-for-cargo-features/